include/boost/geometry/algorithms/detail/disjoint/multipoint_geometry.hpp

```diff
@@ -108,7 +109,7 @@ class multipoint_multipoint
         using predicate_type = unary_not_disjoint_predicate
             <
                 typename std::vector<point1_type>::const_iterator,
-                cs_tag
+                Strategy
```

Comment:
This (for example) looks like a breaking change. Can you point if there is something which could (potentially) cause non compilations for existing users?

Reply:
All strategies (apart from convex hull strategy) calls the old code at the end so AFAIU there is no issue at this point. But there are breaking changes in this PR (see below).